### PR TITLE
modify docs to refer to mainnet

### PR DESCRIPTION
**Pull request by 2dpetkov**

### CODE REVIEW

**Review by dydxoperationstrust — APPROVED** [review]

File: pages/_meta.json [hunk 1]
```diff
@@ -2,15 +2,15 @@
...
   "integration_docs": "Integration documents",
   "developer_links": "Developer links",
   "governance": "Governance",
   "operational_checklist": "Operational checklist",
   "app": {
-    "title": "Testnet",
+    "title": "Mainnet",
     "type": "page",
-    "href": "https://v4.testnet.dydx.exchange",
+    "href": "https://v4-mainnet-docs.vercel.app/",
```
Comment (pswies) [inline pages/_meta.json:13]: Don't we want a custom domain for that? Relying on Vercel to this degree seems like a huge vendor lock-in.

File: pages/general/_meta.json [hunk 2]
```diff
@@ -1,7 +1,4 @@
 {
   "official_links": "Official links",
-  "expectations": "Testnet expectations",
-  "terms_of_use_and_privacy_policy": "Terms of Use and Privacy Policy",
-  "rate_limits": "Rate Limits",
-  "other_limits": "Other Limits"
+  "terms_of_use_and_privacy_policy": "Terms of Use and Privacy Policy"
```
Comment (pswies) [inline pages/general/_meta.json:3]: Could you share some reference docs for this JSON, describing the format that we should follow?

File: pages/mainnet/schedule.md [hunk 3]
```diff
@@ -0,0 +1,10 @@
+## Mainnet `dydx-mainnet-1` Schedule:
+
+| Date                 | Activity                                                        |
+| -------------------- | --------------------------------------------------------------- |
+| 023/09/27 12:00 UTC	| 🌉 Start migrating `DYDX` using `wethDYDX` smart contract
```
Comment (pswies) [inline pages/mainnet/schedule.md:5]: Typo in year number.

File: pages/validators/preparing_for_genesis.md [hunk 4]
```diff
@@ -67,35 +50,33 @@ cosmos_sdk_version: v0.47.3
...
+export CHAIN_ID=dydx-mainnet-1
 # Init the home directory
-dydxprotocold init --chain-id=$CHAIN_ID --home=$HOME_TESTNET_3 $DYDX_MONIKER
+dydxprotocold init --chain-id=$CHAIN_ID --home=$HOME_MAINNET_1 $DYDX_MONIKER
 ```
 
 ## Saving/Recovering Consensus Keys
 
-Be sure to **make a copy** of the key pair json `priv_validator_key.json` and `node_key.json` under `$HOME_TESTNET_3/config`, as these key pairs are generated during `dydxprotocold init ....` and **cannot** be recovered later unless they were explicitly derived through the mnenomics.
+Be sure to **make a copy** of the key pair json `priv_validator_key.json` and `node_key.json` under `$HOME_MAINNET_1/config`, as these key pairs are generated during `dydxprotocold init ....` and **cannot** be recovered later unless they were explicitly derived through the mnenomics.
```
Comment (pswies) [inline pages/validators/preparing_for_genesis.md:73]: "mnenomics": typo? or I don't know the word.

**Review by pswies — COMMENTED** [review]
Added a few comments.

File: pages/_meta.json (hunk 1 above)
Comment (2dpetkov) [reply pages/_meta.json:13]: Certainly, when we have the domain ready, this should be updated. 
Actually, I'll put https://dydx.trade here, not to postpone.
Your commend, however :) is very good! This should link to the exchange FE, not to the documentation.

Will fix in PR#3

File: pages/general/_meta.json (hunk 2 above)
Comment (2dpetkov) [reply pages/general/_meta.json:3]: as far as I can see, the key is the name of the `md` file, the value is the display name. Haven't read the docs

File: pages/mainnet/schedule.md (hunk 3 above)
Comment (2dpetkov) [reply pages/mainnet/schedule.md:5]: Good catch!

File: pages/validators/preparing_for_genesis.md (hunk 4 above)
Comment (2dpetkov) [reply pages/validators/preparing_for_genesis.md:73]: it's a typo 🙏

File: pages/general/_meta.json (hunk 2 above)
Comment (pswies) [reply pages/general/_meta.json:3]: Ah, cool, I actually missed the fact that the `terms_of_use_and_privacy_policy` key was there before, I thought you added it now :)

**Pull request merged**